### ISSUE TITLE
Improve visibility of bars

### ISSUE THREAD

**Issue by larsen0815** [skx/puppet-summary#90]
In the "Node Summary" view, there are bars on top representing the last puppet runs and their states. As the colors used are quite pale, it is hard to distinguish between the states, especially when one state has only a small percentage.

If this could be changed to the colors used when hovering the mouse over a bar, this would be better. I don't see a need for having different colors on hovering. Another way would be to make the colors configurable via a config file.

**Comment by skx** [comment]
This project is basically "done".  I might make minor changes if it fails to build/install properly under modern-go, but short of that I don't use it any more, and it's not been touched for a long time.

If you were able to make changes that were simple and made it look nicer then I'd accept them.  But tweaking colours feels like a personal-preference thing - for any change you made that made it look nicer/better to you, I'm sure somebody else would complain.

So I'd imagine this has to be a very low priority..

**Comment by larsen0815** [comment]
No, haven't made any changes so far. But thanks for your feedback!
